### PR TITLE
Add FrameworkDetails tab to view and edit basic framework information

### PR DESCRIPTION
**Pull request by samshara**

### CODE REVIEW

File: src/views/AnalyticalFramework/FrameworkDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const {
+        pending: frameworkGetPending,
+        response: analyticalFramework,
+    } = useRequest<AnalyticalFramework>({
+        skip: isNotDefined(frameworkId),
+        url: `server://analysis-frameworks/${frameworkId}/`,
+        method: 'GET',
+        onFailure: (_, errorBody) =>
+            notifyOnFailure(_ts('analyticalFramework', 'title'))({ error: errorBody }),
+    });
```
Comment (AdityaKhatri) [inline src/views/AnalyticalFramework/FrameworkDetails/index.tsx:31]: Lets fetch this in AnalyticalFramework view as its values will be used in other tabs as well.
If we do that, we can directly place the Form component outside and we can remove one level of nesting.

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]
Fix review requests

File: src/views/AnalyticalFramework/FrameworkDetails/index.tsx (hunk 1 above)
Comment (samshara) [reply src/views/AnalyticalFramework/FrameworkDetails/index.tsx:31]: Let's do this when it is necessary. We need to add Framework's users table too in a later phase so the nesting of Form component makes sense here.

File: src/views/AnalyticalFramework/FrameworkDetails/FrameworkDetailsForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                            className={styles.createdOn}
+                            name="createdAt"
+                            value={value.createdAt?.split('T')[0]}
+                            readOnly
+                            label={_ts('analyticalFramework', 'createdOn')}
+                        />
+                    </div>
+                    <SelectInput
+                        name="organization"
+                        options={[]}
```
Comment (tnagorra) [inline src/views/AnalyticalFramework/FrameworkDetails/FrameworkDetailsForm/index.tsx]: Is this a work in progress?

File: src/views/AnalyticalFramework/FrameworkDetails/FrameworkDetailsForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        onValueChange,
+        validate,
+        onValueSet,
+        onErrorSet,
+    } = useForm(analyticalFrameworkFromProps ?? defaultFormValues, schema);
+
+    const {
+        pending: frameworkPatchPending,
+        trigger: patchFramework,
+    } = useLazyRequest<AnalyticalFramework, unknown>({
```
Comment (tnagorra) [inline src/views/AnalyticalFramework/FrameworkDetails/FrameworkDetailsForm/index.tsx]: The second type should not be unknown

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Pull request merged**